### ISSUE TITLE
Product view page scrolls up randomly on mobile device

### ISSUE THREAD

**Issue by xiaolinwuu** [magento/magento2#21717]
In the product view page when I scroll down a little bit, sometimes (very often) the page scrolls to the top.

### Preconditions (*)
1. Magento 2.3.0 (in my case) but I think also in previous versions
2. Physical mobile device, Product view page

### Steps to reproduce (*)
1. Go to any product view page
2. Scroll down slowly a little bit like 20-30px
3. If it can't reproduces retry again

### Expected result (*)
1. The page should not scroll up automatically

### Actual result (*)
1. The page scrolls up randomly

This is the effect:
https://www.youtube.com/watch?v=aKwLVwIMrpM

**Comment by magento-engcom-team** [comment]
Hi @xiaolinwuu. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me 2.3-develop instance` - upcoming 2.3.x release

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@xiaolinwuu do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by xiaolinwuu** [comment]
I found out the scroll up takes place only on mobile chrome application, I think the issue is caused by the chrome topbar that shows/hides when we manually scroll the page

**Comment by xiaolinwuu** [comment]
The issue was in the two elements of the fotorama widget:
.fotorama__active
.fotorama__img

They have top: 50% and position: absolute, so when chrome topbar shows/hides, the page suddenly jumps to the top.
 

I solved the problem extending the following css:

```
@media @mobile {
        .catalog-product-view {
            .fotorama__stage__frame.fotorama__active {
                position: static !important;
            }
            .fotorama__stage__frame .fotorama__img {
                position: static !important;
                transform: initial !important;
            }
       }
}
```

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

-  _Next steps are available in case you are a member of **Community Maintainers**._

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by shikhamis11** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11, here is your [Magento instance](https://i-21717-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-21717-2-3-develop.instances.magento-community.engineering/admin](https://i-21717-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
@shikhamis11 Thank you for verifying the issue.

Unfortunately, not enough information was provided to acknowledge ticket. Please consider adding the following:
- [ ] Add `"Component: "` label(s) to this ticket based on verification result. If uncertain, you may follow the best guess

Once all required information is added, please add label `"Issue: Confirmed"` again.
Thanks!

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @shikhamis11 
 Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-98944`, `MAGETWO-98945` were created

**Issue Available**: @shikhamis11, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by dev-ajaysaini** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @webkul-ajaysaini. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @webkul-ajaysaini, here is your [Magento instance](https://i-21717-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-21717-2-3-develop.instances.magento-community.engineering/admin](https://i-21717-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by krzksz** [comment]
Closing this as a dupplicate of https://github.com/magento/magento2/issues/10518

**Comment by slavvka** [comment]
Hi @xiaolinwuu. Thank you for your report. 
The issue has been fixed in magento/magento2#25385 by @krzksz in 2.4-develop branch
Related commit(s):
 - [0b278c84278cb42f03083a4115763d3ed5c7dd52](https://github.com/magento/magento2/commit/0b278c84278cb42f03083a4115763d3ed5c7dd52)
 - [d74f73c7272887c68d595fafd25d997c815758e1](https://github.com/magento/magento2/commit/d74f73c7272887c68d595fafd25d997c815758e1)
 - [80b8514a601923b66abb8492e0868ad7fb5dfb1f](https://github.com/magento/magento2/commit/80b8514a601923b66abb8492e0868ad7fb5dfb1f)
 - [c5991d3c719d68f66c64f5f10978b6412021165c](https://github.com/magento/magento2/commit/c5991d3c719d68f66c64f5f10978b6412021165c)
 - [acdc9b429bfab2a9aa496f571d9ccc37d0b6492d](https://github.com/magento/magento2/commit/acdc9b429bfab2a9aa496f571d9ccc37d0b6492d)

The fix will be available with the upcoming 2.4.0 release.
